### ISSUE TITLE
Library documentation build by OpenModelica broken

### ISSUE THREAD

**Issue by maltelenz** [modelica/www.modelica.org#82]
https://modelica.org/libraries/ links to https://build.openmodelica.org/Documentation/ very prominently, but images and styling are broken on the latter page:

![image](https://github.com/modelica/www.modelica.org/assets/329963/ccf1a6e3-e612-40df-835c-6a61925a5444)

**Comment by beutlich** [comment]
Is this an OMC issue?

**Comment by maltelenz** [comment]
There are two issues, I guess:

1. The OpenModelica builds are broken.
2. modelica.org prominently links to something that is broken.

**Comment by dzimmer** [comment]
As long as OpenModelica builds is broken, we shall remove the link

**Comment by casella** [comment]
Yes, sorry. I opened OpenModelica/OpenModelica#12693 to get that fixed.
